src/Symfony/Component/Security/Http/Firewall/ContextListener.php

```diff
@@ -268,4 +274,12 @@ public static function handleUnserializeCallback($class)
     {
         throw new \UnexpectedValueException('Class not found: '.$class, 0x37313bc);
     }
+
+    /**
+     * @internal
+     */
+    public function setRememberMeServices(RememberMeServicesInterface $rememberMeServices)
```

Comment:
can this be turned into a constructor argument? that'd be better

Reply:
It can but would require to add an extra `$contextListenerId` argument in RembermeFactory::create() which is interfaced).
This is the easy way: RememberMeFactory adds this method call to all `security.remember_me_aware` tagged services, other core listeners rely on it (e.g. guard).
Worth changing?

Reply:
Works for me, thanks for the explanation.